source/math/FloatVectorList.ooc

```diff
@@ -181,16 +186,10 @@ FloatVectorList: class extends VectorList<Float> {
 		max != 0 ? (this / max) : this copy()
 	}
 	getOnes: static func (count: Float) -> This {
-		result := This new(count)
-		for (i in 0 .. count)
-			result add(1.0f)
-		result
+		This new(count, 1.0f)
 	}
 	getZeros: static func (count: Float) -> This {
-		result := This new(count)
-		for (i in 0 .. count)
-			result add(0.0f)
-		result
+		This new(count, 0.0f)
```

Comment:
Ah yes. I'm not sure there's a net save in terms of lines of code, but sure. Done.